Library/Homebrew/formula.rb

```diff
@@ -41,6 +42,7 @@
 class Formula
   include FileUtils
   include Utils::Inreplace
+  include Utils::Shell
```

Comment:
You shouldn't need to do this anymore if you're having `#shell_profile` call to the fully-qualified name.